python/tests/test_aac/plugins/gen_plugin/test_gen_plugin.py

```diff
@@ -51,36 +41,112 @@ def test_cli_gen_plugin(self):
-            self.assertTrue(os.path.exists(os.path.join(package_src_path, "my_plugin_impl.py")))
-            self.assertTrue(os.path.exists(os.path.join(package_tests_path, "test_my_plugin.py")))
-            self.assertTrue(os.path.exists(os.path.join(package_tests_path, "my_plugin_command_test.feature")))
-            self.assertTrue(os.path.exists(os.path.join(package_tests_path, "my_plugin_context_test.feature")))
-            self.assertTrue(os.path.exists(os.path.join(package_tests_path, "my_plugin_schema_test.feature")))
-            self.assertTrue(os.path.exists(os.path.join(package_tests_path, "my_plugin_primitive_test.feature")))
+            self.assertTrue(path.exists(path.join(package_src_path, "__init__.py")))
+            self.assertTrue(path.exists(path.join(package_src_path, "my_plugin_impl.py")))
+            self.assertTrue(path.exists(path.join(package_tests_path, "test_my_plugin.py")))
+            self.assertTrue(path.exists(path.join(package_tests_path, "my_plugin_command_test.feature")))
+            self.assertTrue(path.exists(path.join(package_tests_path, "my_plugin_context_test.feature")))
+            self.assertTrue(path.exists(path.join(package_tests_path, "my_plugin_schema_test.feature")))
+            self.assertTrue(path.exists(path.join(package_tests_path, "my_plugin_primitive_test.feature")))
+
+    def test_cli_gen_plugin_overwrite(self):
+        # first we need a project to work in, so generate a temporary one
+        with TemporaryDirectory() as temp_dir:
+            # now create an AaC plugin file in the project src directory
+            aac_file_path = path.join(path.dirname(__file__), "my_project.aac")
+            temp_aac_file_path = path.join(temp_dir, "my_project.aac")
+            copy(aac_file_path, temp_aac_file_path)
+
+            proj_args = [temp_aac_file_path, "--output", temp_dir, "--no-prompt"]
+            exit_code, output_message = self.run_gen_project_cli_command_with_args(proj_args)
+
+            package_src_path = path.join(temp_dir, "src", "happy")
+            mkdir(package_src_path)
+            package_tests_path = path.join(temp_dir, "tests", "test_happy")
+            mkdir(package_tests_path)
+            plugin_file_path = path.join(package_src_path, "my_plugin.aac")
+
+            aac_plugin_path = path.join(path.dirname(__file__), "my_plugin.aac")
+            copy(aac_plugin_path, plugin_file_path)
+            plugin_args = [plugin_file_path, "--code-output", path.join(temp_dir, "src"), "--test-output", path.join(temp_dir, "tests"), "--no-prompt"]
+            exit_code, output_message = self.run_gen_plugin_cli_command_with_args(plugin_args)
+            file = open(path.join(package_src_path, "my_plugin_impl.py"), "r")
+            file_read = file.read()
+            self.assertIn("test_command_one(aac_plugin_file: str)", file_read)
+            self.assertNotIn("test_command_one_evaluation(aac_plugin_file: str)", file_read)
```

Comment:
```suggestion
            self.assertNotIn("test_command_1_evaluation(aac_plugin_file: str)", file_read)
```
Based on the behavior I am seeing when I execute this workflow manually, the expected output for the evaluate method name has 1 rather than one.